packages/internal/src/__tests__/build_api.test.ts

```diff
@@ -93,13 +94,186 @@ test.skip('api prebuild transforms gql with `babel-plugin-graphql-tag`', () => {
 
 test('Pretranspile polyfills unsupported functionality', () => {
   const p = prebuiltFiles.filter((p) => p.endsWith('polyfill.js')).pop()
+
   const code = fs.readFileSync(p, 'utf-8')
+
+  expect(code).toContain(
+    `import _AggregateError from "core-js-pure/features/aggregate-error.js";`
+  )
```

Comment:
Commenting on this file generally; note that the diff isn't too useful till https://github.com/redwoodjs/redwood/pull/6208 gets merged. So I wouldn't focus on this just yet.